platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/crypto/CryptoArgsProvider.java

```diff
@@ -39,23 +44,22 @@ public class CryptoArgsProvider {
      */
     static Stream<Arguments> basicTestArgs() throws Exception {
         Instant start = Instant.now();
-        final AddressBook loadedAB = createAddressBook();
-        final Map<NodeId, KeysAndCerts> loadedC =
-                CryptoStatic.loadKeysAndCerts(loadedAB, ResourceLoader.getFile("preGeneratedKeysAndCerts/"), PASSWORD);
+        final Pair<AddressBook, Map<NodeId, KeysAndCerts>> addressBookWithKeys =
+                loadAddressBookWithKeys(NUMBER_OF_ADDRESSES);
```

Comment:
I suggest avoiding the `Pair<>` type, and to instead create a simple record. Records are simple to create and create much more readable code.

Reply:
Preferences :) I actually *really do* happen to like the Pair construct more - they read exactly like what they do, hold `a pair of things`. No strong preference so I'll remove.